addons/account_peppol/wizard/account_move_send.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from base64 import b64encode
+
+from odoo import api, fields, models, _
+from odoo.addons.account_edi_proxy_client.models.account_edi_proxy_user import AccountEdiProxyError
+from odoo.exceptions import UserError
+
+
+class AccountMoveSend(models.Model):
+    _inherit = 'account.move.send'
+
+    checkbox_send_peppol = fields.Boolean(
+        string='Send via PEPPOL',
+        help='Send the invoice via PEPPOL',
+    )
```

Comment:
Follow the same logic as the others modules. You need a field to enable or not the checkbox and this one allowing the user to enable Peppol.